CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.0.4] - 2021-07-22
```

Comment:
You also need to add a line at the bottom of the file to make this `[1.0.4]` link work.

Reply:
Got it, and updated. Thanks!

Reply:
Oops - missed the Unreleased link update. One sec.

Reply:
OK, all set now.